open-company/open-company-web
# Fix shaky again

Card: https://trello.com/c/susDKdVV

Digest url: /11-october-5/all-posts?at=2018-10-12T13:13:07.432Z
the at parameter is the published-at of the post

To test:
- check locally with a digest URL of the company for whom you are logged in
- [x] do you see the post? Good
- use the same url but change the timestamp to be fake (ie: not found)
- [x] do you see the post removed screen? Good
- check locally with a digest URL of another org
- [x] do you see the post removed wall? Good
- now logout
- check a digest URL (not a public section)
- [x] do you see the login wall? Good
- try to login using one of the 3 methods
- deploy and check the same on staging (timing of the resource load is different, that may affect the check even if it shouldn't)

TODO: remember the URL and redirect there after login, right now it's not a quick thing, added a backlog card for it
